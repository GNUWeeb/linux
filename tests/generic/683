#! /bin/bash
# SPDX-License-Identifier: GPL-2.0
# Copyright (c) 2022 Christian Brauner (Microsoft).  All Rights Reserved.
#
# FS QA Test No. 683
#
# Test that setting POSIX ACLs in userns-mountable filesystems works.
#
# Regression test for commit:
#
# 705191b03d50 ("fs: fix acl translation")
#
. ./common/preamble
_begin_fstest auto quick perms

# Import common functions.
. ./common/filter

# real QA test starts here

_supported_fs generic
_require_test
_require_user fsgqa
_require_group fsgqa

echo "Silence is golden"

$here/src/idmapped-mounts/idmapped-mounts --test-setxattr-fix-705191b03d50 \
	--device "$TEST_DEV" --mount "$TEST_DIR" --fstype "$FSTYP"

status=$?
exit
