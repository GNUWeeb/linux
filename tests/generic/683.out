QA output created by 683
Silence is golden
